tests/transaction_log_parsing.cpp

```diff
@@ -1325,8 +1325,8 @@ TEST_CASE("Transaction log parsing: changeset calcuation") {
 
         auto observe = [&](std::initializer_list<Row> rows, auto&& fn) {
             auto history = make_in_realm_history(config.path);
-            SharedGroup sg(*history, config.options());
-            auto& group = sg.begin_read();
+            std::unique_ptr<SharedGroup> sg = std::make_unique<SharedGroup>(*history, config.options());
```

Comment:
This is a good place to use `auto` as otherwise the type of the variable is specified twice.